api/regen/data/v1/events.pulsar.go

```diff
@@ -1399,7 +1399,7 @@ func (x *fastReflection_EventStoreRawData) ProtoMethods() *protoiface.Methods {
 // Code generated by protoc-gen-go. DO NOT EDIT.
 // versions:
 // 	protoc-gen-go v1.27.0
-// 	protoc        v3.19.1
```

Comment:
im not a docker expert, but im wondering if maybe protoc isn't installed in the container? https://grpc.io/docs/protoc-installation/

that protoc version in the -diff is the one i had installed

Reply:
Hmm... protoc is installed as it was on my system. I'm using a later version.

```
$ protoc --version
libprotoc 3.19.4
```

Reply:
I think this is a minor detail and the sdk is using the same setup and generates `(unkown)` as well. Would be nice to get this merged so that we are all generating the same output. Updating the tendermint container is also out of scope for this pull request and not something we can do.

Reply:
if protoc wouldn't be installed then we wouldn't be able to generate anything, right?